app/create_app.py

```diff
@@ -106,10 +108,11 @@ def inject_global_constants():
     def inject_service_name():
         fund = None
         try:
-            if request.view_args.get("fund_short_name"):
-                fund = get_fund_data_by_short_name(
-                    request.view_args.get("fund_short_name")
-                )
+            fund_short_name = request.view_args.get("fund_short_name")
+            if fund_short_name and str.upper(fund_short_name) in [
```

Comment:
Do you think an assignment expression might be better here? It'll kill the variable context sooner.

https://peps.python.org/pep-0572/

Reply:
that doc is quite long so this being the specific example :

if (match := pattern.search(data)) is not None:
    # Do something with match